File: Modul-2/Soal-4/PRAK204-2410817220023-CLARISSADHEAALLISYA.py
# test case 1
radius = float(input())
tinggi = float(input())

volume = 22 / 7 * radius * radius * tinggi
luas = 2 * 22 / 7 * radius * (radius + tinggi)
keliling = 2 * 22 / 7 * radius

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
print("")

# test case 2
radius2, tinggi2 = map(float, input().split())

volume2 = 22 / 7 * radius2 * radius2 * tinggi2
luas2 = 2 * 22 / 7 * radius2 * (radius2 + tinggi2)
keliling2 = 2 * 22 / 7 * radius2

print(f"Volume = {volume2:.2f}")
print(f"Luas = {luas2:.2f}")
print(f"Keliling = {keliling2:.2f}")

File: Modul-2/Soal-4/PRAK204-2410817220023-CLARISSADHEAALLISYA.c
#include <stdio.h>

int main(){
    // test case 1
    float radius, tinggi, volume, luas, keliling;
    scanf("%f", &radius); 
    scanf("%f", &tinggi);

    volume = radius * radius * tinggi * 22/7;
    luas= 2 * radius * (radius + tinggi) * 22/7;
    keliling= 2 * radius * 22/7;

    printf("Volume = %.2f \n", volume);
    printf("Luas = %.2f \n", luas);
    printf("Keliling = %.2f \n", keliling);
    printf("\n");

    // test case 2
    float radius2, tinggi2, volume2, luas2, keliling2;
    scanf("%f", &radius2); scanf("%f", &tinggi2);

    volume2 = radius2 * radius2 * tinggi2 * 22/7;
    luas2 = 2 * radius2 * (radius2 + tinggi2) * 22/7;
    keliling2 = 2 * radius2 * 22/7;

    printf("Volume = %.2f \n", volume2);
    printf("Luas = %.2f \n", luas2);
    printf("Keliling = %.2f \n", keliling2);
    return 0;
}